### PR TITLE
16 phase1 prompting

### PR DESCRIPTION
**Pull request by Loxerate**
I Think I addressed all the comments from the previous PR. Thank you for Reviewing this Jie.

### CODE REVIEW

File: generate_response.py [hunk 1]
```diff
@@ -1051,7 +1081,8 @@ def HumanEval_experiment(dataset, dataset_loc, option, model, topn, temperature,
                     # We will use "prompt_modified" to check whether AgentCoder is getting a modified prompt or an original prompt, based on which, we decide whether to send in a "generate clarifying questions" prompt or not.
                     # A new prompt called PROMPT_START_3_v4 has been created for the same.
                     prompt_modified = False if input_prompt == 'prompt' else True
-                    prompt_start = ORIGINAL_PROMPT_START_0 if input_prompt == 'prompt' else PROMPT_START_3_v2
+                    #prompt_start = ORIGINAL_PROMPT_START_0 if input_prompt == 'prompt' else PROMPT_START_3_v2
```
Comment (jie-jw-wu) [inline generate_response.py:1084]: delete commented code

File: generate_response.py [hunk 2]
```diff
@@ -643,6 +653,7 @@ def create_prompt(description, option='original', percentage=0):
     elif option.startswith('randRemove'):
         return PROMPT_START_3 + split_and_remove_chunk(description, percentage)
     elif option.startswith('manualRemove'): # TODO(jwu): WIP
+        #return PROMPT_START_3_v2 + description
```
Comment (jie-jw-wu) [inline generate_response.py:656]: delete commented code

**Review by jie-jw-wu — APPROVED** [review]
approved w/ comments. please test it to ensure it's correct. thanks!

**Comment by Loxerate** [comment]
@jie-jw-wu The problem with the prompt argument was that I had only updated the .sh file and not the .bat file. Now it works as it should.

**Pull request merged**